detox/e2e/support/server_api/plugin.ts

```diff
@@ -45,7 +78,7 @@ export const apiDisableNonPrepackagedPlugins = async (baseUrl: string): Promise<
         return;
     }
     plugins.active.forEach(async (plugin: any) => {
-        if (!prepackagedPlugins.has(plugin.id)) {
+        if (plugin.id !== DemoPlugin.id && !prepackagedPlugins.has(plugin.id)) {
```

Comment:
@yasserfaraazkhan - Since we are running them all at once, we don't want to keep this do we?